.github/workflows/almalinux.yml

```diff
@@ -0,0 +1,18 @@
+name: AlmaLinux
+on:
+  push:
+    paths:
+      - 'install_ant-media-server.sh'
+jobs:
+  Almalinux8:
+    runs-on: ubuntu-20.04
+    container: almalinux:8
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/deploy-almalinux
```

Comment:
This install the latest version and deploy-almalinux-9 installs the latest snapshot. Please make the names self-explanatory Otherwise it makes life hard for the other contributor.

Check the self-explanatory workflows as below

Install Latest Snapshot to Ubuntu 22.04
Install Latest to Ubuntu 20.04
Install v2.4 to Ubuntu 20.04